env/dev/jun19_fairroot_18_4/spack.yaml

```diff
@@ -1,26 +1,30 @@
 spack:
   definitions:
-    - when: platform == 'linux'
-      root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite ^mesa~llvm]
-    - when: platform == 'darwin'
-      root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite+aqua]
+  - when: platform == 'linux'
+    root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite
+        ^mesa~llvm]
+  - when: platform == 'darwin'
+    root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite+aqua]
+  - concretizer_hints:
+    - python@2.7.18
+    - py-numpy@1.16.6
+    - py-setuptools@44.1.0
   specs:
-    - pcre+jit
-    - python@2.7.16
-    - py-numpy@1.16.5
-    - googletest@1.8.1
-    - boost@1.68.0
-    - fairlogger@1.4.0
-    - dds@2.4
-    - fairmq@1.4.3
-    - pythia6@428-alice1
-    - hepmc@2.06.09 length=CM momentum=GEV
-    - pythia8@8240
-    - geant4@10.05.p01~qt~vecgeom~opengl~x11~motif~data~clhep+threads
-    - $root
-    - geant3@2-7_fairsoft
-    - vgm@4-5
-    - geant4-vmc@4-0-p1
-    - fairroot@18.4.0+sim+examples
+  - $concretizer_hints
+  - fairsoft-config@jun19
```

Comment:
As the `jun19_fairroot_18_4` env is mainly for CI (maybe should be moved to `test/`?), why do you need to add fairsoft-config? Or does that fairoot-Version (18.4) really need it?

Reply:
We can move the envs to test in a separate PR.